buildSrc/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
 	}
 }
 
-ext.xtextVersion = '2.10.0'
+ext.xtextVersion = '2.17.0'
```

Comment:
bump the xtend plugin too. version is 2.0.4
maybe bump gradle version (wrapper) as well

Reply:
gradle wrapper is bumped, or do you mean sthg else?

Reply:
did not see the wrapper. but the plugin is the old

Reply:
already changed, see new commit

Reply:
did you run the wrapper command twice?

Reply:
(would have expected changes to sh and bat file too